### DISCUSSION TITLE
Client support for versions after 2021E

### DISCUSSION THREAD

**Discussion by Connorcivi** [Windows81/Roblox-Freedom-Distribution#26] [Ideas]
Hello again,

So I've been doing some stuff in the background for some other stuff and just got back to this. I found out that Attributes are not supported for the 2021/2018 versions. Is there any chance that we can get Attribute support?

**Comment by Windows81** [comment]
Version 463 is the last version of which Rōblox released their server-side
server for Windows.

As for attributes, refer to #13.

There is a flag that I should've already enabled.  And that flag allows attributes to work.

**Comment by Connorcivi** [comment]
Okay, I'll update the roblox client and server later today. Um, I do want to address that RFD.exe is coming up as a virus on windows, and I get it, it's python's stupid thingy. But if you could look into it, that'd be awesome.

**Comment by Connorcivi** [comment]
I did try to make my own "Version" by using the existing code n trying to make an exe out of it but python is just being janky as it keeps reporting a virus everytime.

**Comment by Windows81** [comment]
I might've acknowledged that somewhere on the Rōblo Devforum.  You're more than welcome to edit the README or devise a better way to compile the codebase to an EXE.

**Comment by Windows81** [comment]
Just disable your anti-virus.  If you can't do that, just re-install Windows.

**Comment by Connorcivi** [comment]
Ok so, I enabled Arttibutes, but the collisiongroups are not anywhere to be found. Is there any chance that it can be a requested feature to be added?

**Comment by Connorcivi** [comment]
![image](https://github.com/user-attachments/assets/7f5cad71-4c70-49a6-a70d-c63c4a79bcac)

This is what I mean btw.

**Comment by Windows81** [comment]
`RegisterCollisionGroup` didn't exist unti 2022M.

https://robloxapi.github.io/ref/class/PhysicsService.html#member-RegisterCollisionGroup

**Comment by Connorcivi** [comment]
Ah understandable, is there a guide where we can upload custom versions or
is it just not viable?

**Comment by Windows81** [comment]
RFD is designed to run with a particular executable named RCC.  The last available build for RCC that we have is v463.

People have got client version ~483 to work with the RCC that I have.

If you want to run Rōblox servers from after v463, you'd use Rōblox Studio.  RFD's guides do not cover Studio.

Nothing is impossible.  If you need clarification or my language is unclear, feel free to ask me.

**Comment by Cuphead74832** [comment]
Hey for anyone interested.
https://github.com/Roblox-Devs/studio-offline/tree/main
This sounds interesting.... I may try It. Althrough last update was 9 months ago...

**Reply by Cuphead74832** [reply]
I tried the project, its pretty outdated so It doesnt work anymore, so we have to wait for someone to revamp it.
